test/math/FloatPoint2DTest.ooc

```diff
@@ -49,9 +50,8 @@ FloatPoint2DTest: class extends Fixture {
 		this add("casting", func {
 			value := "10.00, 20.00"
 			expect(this point3 toString(), is equal to(value))
-//			FIXME: Equals interface
-			expect((FloatPoint2D parse(value)) x, is equal to((this point3) x))
-			expect((FloatPoint2D parse(value)) y, is equal to((this point3) y))
+			expect((FloatPoint2D parse(t"10.00,20.00")) x, is equal to((this point3) x))
```

Comment:
Maybe `toFloat` is smart enough that it can discard the space by itself... But alright. Assign `t"10.00,20.00"` to a variable at least, so you don't have to define it twice.